### PR TITLE
Advanced view for files

### LINKED ISSUE #442: Advanced files view and download

### PR DESCRIPTION
**Pull request by ArsalanSavand**
Fixes #442 

- Modal view when hovered
image

- List view
image

### CODE REVIEW

File: src/app/dolphin/dolphin_modal/dolphin_modal_view.html [hunk 1]
```diff
@@ -1,40 +1,70 @@
...
-        <!-- Form -->
-        <form ng-submit="update(form)" name="form">
-          <div class="thumbnail" ng-if="form.data.is_image">
-            <img ng-src="{{ form.data.file }}" alt="form.data.meta_data.name">
-          </div>
-          <div class="form-group" ng-class="{'has-error': form.errors.meta_data.name}">
-            <input class="form-control" type="text" ng-model="form.data.meta_data.name"
-              placeholder="{{ 'FILE_NAME' | translate }}" maxlength="100" required>
-            <small class="help-block" ng-show="form.errors.file_name">{{ form.errors.meta_data.name[0] }}</small>
+      <!-- <div class="modal-header">
```
Comment (Alir3z4) [inline src/app/dolphin/dolphin_modal/dolphin_modal_view.html]: Remove this commented part if it's not needed anymore.

File: src/app/dolphin/dolphin_modal/dolphin_modal_view.html [hunk 2]
```diff
@@ -1,40 +1,70 @@
...
+        </div>
+
+        <div class="file-overlay" ng-class="{'img': form.data.is_image}">
+          <!-- Name -->
+          <h3 class="s-pad">
+            {{ form.data.meta_data.name }} <i class="fa fa-close o-fade-soft fa-fw pull-right c-p" data-dismiss="modal"></i>
+          </h3>
+          <!-- Info -->
+          <span class="label label-default s-mar-left s-mar-right-soft">
+            <i class="fa fa-fw fa-hdd-o" title="Size"></i> {{ dolphin.get.size_human }}
```
Comment (Alir3z4) [inline src/app/dolphin/dolphin_modal/dolphin_modal_view.html]: Size needs to be marked for translation.

File: src/app/dolphin/dolphin_modal/dolphin_modal_view.html [hunk 3]
```diff
@@ -1,40 +1,70 @@
...
+          </span>
+          <span class="label label-default">
+            <i class="fa fa-fw fa-file-o" title="Extension"></i> {{ dolphin.extension | lowercase }}
```
Comment (Alir3z4) [inline src/app/dolphin/dolphin_modal/dolphin_modal_view.html]: Extension needs to be marked for translation.

**Comment by Alir3z4** [comment]
3 comments on the modal buttons as well

1. Why they have been placed on the left side of the modal ? They usually are on the bottom right side.
2. Why modal buttons are not in an group buttons ?
3. Add a cancel button to the buttons as well.

**Comment by Alir3z4** [comment]
Also make sure you have all the translations in the en.json if you have added or removed any.

**Comment by AmirSavand** [comment]
@Alir3z4 note that screenshots are hover state, if you don't hove them the overlay will disappear.

**Comment by Alir3z4** [comment]
LGTM!

**Pull request merged**